assets/training/model_management/environments/model-management/context/Dockerfile

```diff
@@ -25,7 +25,27 @@ RUN conda install python=3.9 -y
 RUN pip install -r requirements.txt --no-cache-dir
 
 # Vulnerability fix
-RUN pip install torch==2.6.0 torchvision --no-cache-dir
+# Remove Conda-installed Torch and related packages
+RUN conda remove -y --force \
```

Comment:
can you use torch26 image from aifx instead? it defeats the purpose of using that base image